src/frontend/pages/home/Home.css

```diff
@@ -1,24 +1,28 @@
 .home-page h1 {
```

Comment:
Looks like your VS code's changed the tab width in this file to 2 spaces rather than 4 spaces - can you check your configuration/settings and then fix this file to use 4 spaces once more?